mappings/net/minecraft/entity/mob/MobEntityWithAi.mapping

```diff
@@ -3,11 +3,14 @@ CLASS aib net/minecraft/entity/mob/MobEntityWithAi
 	FIELD c aiRadius F
 	METHOD <init> (Lahp;Lbdb;)V
 		ARG 1 type
+	METHOD a getPathfindingFavor (Les;)F
 	METHOD a setAiHome (Les;I)V
 		ARG 1 home
+	METHOD a getPathFindingFavor (Les;Lbde;)F
```

Comment:
getPathfindingFavor to match the one above (and the other names already mapped)

Reply:
This is somewhat unusual because a higher number here means higher priority, unlike the established java natural order for numbers

Reply:
No, no - you capitalized it differently

Reply:
Resolved.